package.xml

```diff
@@ -20,6 +20,7 @@
 
   <build_depend>builtin_interfaces</build_depend>
   <build_depend>rclcpp</build_depend>
+  <build_depend>rclcpp_lifecycle</build_depend>
```

Comment:
Why do you need to depend on `rclcpp_lifecycle` here?

I think you don't use it at all, other than to reference it in the documentation.